### DISCUSSION TITLE
New User - stuck at "quire preview"

### DISCUSSION THREAD

**Discussion by metro-glenn** [thegetty/quire#724] [Get Help] [answered]
Hello! I'm giving Quire a try on a Windows 11 PC. So far:

1. successfully installed git version 2.40.1.windows.1
2. successfully installed node v18.16.0
3. successfully installed quire 1.0.0-rc.7
4. successfully created a new project with "quire new fashion-reperformances"
5. and changed to that directory

But then when I run "quire preview" I get:

PS C:\Users\glenn\fashion-reperformances> quire preview
[CLI] running eleventy using lib/11ty cli
[CLI:11ty] running eleventy serve
node:internal/modules/cjs/loader:1078
  throw err;
  ^

Error: Cannot find module 'C:\Users\glenn\fashion-reperformances\node_modules\@11ty\eleventy\cmd.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1075:15)
    at Module._load (node:internal/modules/cjs/loader:920:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:23:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v18.16.0
PS C:\Users\glenn\fashion-reperformances>

I'm not sure how to proceed from here. Thank you for any help!

**Comment by Erin-Cecele** [comment]
Hi @metro-glenn. Thank you for posting this question. Can you please run the command `npm install` while in your `fashion-repreformances` folder and then try running `quire preview` again once install is complete? Let us know if that does the trick!

**Reply by metro-glenn** [reply]
Thanks, @Erin-Cecele !
I gave it a try. Same results:

PS C:\Users\glenn> cd fashion-reperformances
PS C:\Users\glenn\fashion-reperformances> npm install

up to date, audited 1 package in 414ms

found 0 vulnerabilities
PS C:\Users\glenn\fashion-reperformances> quire preview
[CLI] running eleventy using lib/11ty cli
[CLI:11ty] running eleventy serve
node:internal/modules/cjs/loader:1078
  throw err;
  ^

Error: Cannot find module 'C:\Users\glenn\fashion-reperformances\node_modules\@11ty\eleventy\cmd.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1075:15)
    at Module._load (node:internal/modules/cjs/loader:920:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:23:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v18.16.0
PS C:\Users\glenn\fashion-reperformances>

**Reply by Erin-Cecele** [reply]
Hi @metro-glenn, let us look into this a little further for you. Thank you for your patience.

**Reply by geealbers** [reply]
@metro-glenn, we've been testing this on our end and so far have been unable to generate the same errors you're seeing. Can I ask you for a little more information? On you machine, can you please go to Settings and scroll down to the About section. There should be some Windows specifications listed there. Can you let us know what it says there (a screenshot is fine)?

Also, can you open your `fashion-reperformances` project in File Explorer, open the `node_modules` folder, and then take a screenshot of that for us? We don't need to be able to see the full contents of that folder, just however many items fit in a single screenshot.

thank you!

**Reply by metro-glenn** [reply]
Hello, @geealbers, thanks for your help!
Here's the system info:
https://gyazo.com/4149591ac5277b5dfdca34a1de42a0fc

And there doesn't seem to be a node_modules in the fashion-reperformances folder:
https://gyazo.com/6e1c7901db2dae7a8dc6e409a105b4cd

**Reply by geealbers** [reply, marked as answer]
@metro-glenn There are quite a number of files missing from your `fashion-reperformances` folder, which indicates the `quire new` command partially failed. I haven't been able to reproduce this on my end though so I don't have a solution. Could you try starting another project and then share with me the complete text output you get from PowerShell? There might be something in there that would give me another clue. And if the folder contents of the new project look different from the `fashion-reperformances` one, another screenshot of that would be useful too. Thanks!

**Reply by metro-glenn** [reply]
Hi @geealbers , just created `fashion-test` and it does have different files:
https://gyazo.com/f302e2be2379bf37c4ae4c69df91fc45

Here's the PowerShell:
---------------------------

Windows PowerShell
Copyright (C) Microsoft Corporation. All rights reserved.

Install the latest PowerShell for new features and improvements! https://aka.ms/PSWindows

PS C:\Users\glenn> quire new fashion-test
[CLI:quire] init-starter
  project root: "fashion-test"
  starter: "https://github.com/thegetty/quire-starter-default"
fashion-test set to use quire-11ty@latest
[CLI:quire] installing @thegetty/quire-11ty@latest into fashion-test
[CLI:quire] installing dev dependencies into quire project
PS C:\Users\glenn>

**Reply by geealbers** [reply]
Ok! The new one looks correct. Can you run `quire preview` on it and see if it works now?

**Reply by metro-glenn** [reply]
Since the fashion-test install installed a lot more files, I tried quire preview again. It seemed to generate a lot of errors, however the preview seems to work:

```
PS C:\Users\glenn> cd fashion-test
PS C:\Users\glenn\fashion-test> quire preview
[CLI] running eleventy using lib/11ty cli
[CLI:11ty] running eleventy serve
[quire] DEBUG    Figures:ImageProcessor
      inputRoot: C:\Users\glenn\fashion-test\content\_assets\images
      outputRoot: C:\Users\glenn\fashion-test\_site

[quire] DEBUG    Figures:ImageProcessor         processing inputPath: C:\Users\glenn\fashion-test\content\_assets\images\figures\lange-mother-a.jpg
[quire] DEBUG    Figures:Figure                 processing annotation image figures/lange-mother-b.jpg
[quire] DEBUG    Figures:ImageProcessor         processing inputPath: C:\Users\glenn\fashion-test\content\_assets\images\figures\lange-mother-b.jpg
[quire] DEBUG    Figures:Figure                 processing annotation image figures/lange-mother-c.jpg
[quire] DEBUG    Figures:ImageProcessor         processing inputPath: C:\Users\glenn\fashion-test\content\_assets\images\figures\lange-mother-c.jpg
[quire] INFO     Figures:IIIF:Manifest          Generated manifest for figure "mother-variants"
[quire] DEBUG    Figures:Figure                 processing annotation image figures/lange-layers/original-label.png
[quire] DEBUG    Figures:ImageProcessor         processing inputPath: C:\Users\glenn\fashion-test\content\_assets\images\figures\lange-layers\original-label.png
[quire] DEBUG    Figures:Figure                 processing annotation image figures/lange-layers/additional-label.png
[quire] DEBUG    Figures:ImageProcessor         processing inputPath: C:\Users\glenn\fashion-test\content\_assets\images\figures\lange-layers\additional-label.png
[quire] DEBUG    Figures:Figure                 processing annotation image figures/lange-layers/scratches.png
[quire] DEBUG    Figures:ImageProcessor         processing inputPath: C:\Users\glenn\fashion-test\content\_assets\images\figures\lange-layers\scratches.png
[quire] DEBUG    Figures:Figure                 processing annotation image figures/lange-layers/thumb.png
[quire] DEBUG    Figures:ImageProcessor         processing inputPath: C:\Users\glenn\fashion-test\content\_assets\images\figures\lange-layers\thumb.png
[quire] DEBUG    Figures:ImageProcessor         processing inputPath: C:\Users\glenn\fashion-test\content\_assets\images\figures\lange-layers\base.jpg
[quire] DEBUG    Figures:ImageProcessor         processing inputPath: C:\Users\glenn\fashion-test\content\_assets\images\figures\evans-fields-family-a.jpg
[quire] DEBUG    Figures:ImageTiler             tiling 'C:\Users\glenn\fashion-test\content\_assets\images\figures\lange-layers\base.jpg'
[quire] DEBUG    Figures:ImageTiler             tiling 'C:\Users\glenn\fashion-test\content\_assets\images\figures\lange-mother-a.jpg'
[quire] DEBUG    Figures:ImageTiler             tiling 'C:\Users\glenn\fashion-test\content\_assets\images\figures\evans-fields-family-a.jpg'
[quire] INFO     Figures:IIIF:Manifest          Generated manifest for figure "mother-annotations"
[quire] INFO     Figures:IIIF:Manifest          Generated manifest for figure "cat-2"
[quire] INFO     Figures:IIIF:Manifest          Generated manifest for figure "cat-1"
[quire] INFO     Figures                        Processing complete
[quire] ERROR    transforms:format              Eleventy transform error formatting output for _site/pdf-epub-title/index.html.
TypeError: import_mockable2.default.findParentDir is not a function
[quire] ERROR    transforms:format              Eleventy transform error formatting output for _site/pdf-epub-half-title/index.html.
TypeError: import_mockable2.default.findParentDir is not a function
[quire] ERROR    transforms:format              Eleventy transform error formatting output for _site/contents/index.html.
TypeError: import_mockable2.default.findParentDir is not a function
[quire] ERROR    transforms:format              Eleventy transform error formatting output for _site/catalogue/index.html.
TypeError: import_mockable2.default.findParentDir is not a function
[quire] ERROR    transforms:format              Eleventy transform error formatting output for _site/about/index.html.
TypeError: import_mockable2.default.findParentDir is not a function
[quire] ERROR    transforms:format              Eleventy transform error formatting output for _site/pdf-epub-copyright/index.html.
TypeError: import_mockable2.default.findParentDir is not a function
[quire] ERROR    transforms:format              Eleventy transform error formatting output for _site/contributors/index.html.
TypeError: import_mockable2.default.findParentDir is not a function
[quire] ERROR    transforms:format              Eleventy transform error formatting output for _site/essay/index.html.
TypeError: import_mockable2.default.findParentDir is not a function
[quire] ERROR    transforms:format              Eleventy transform error formatting output for _site/index.html.
TypeError: import_mockable2.default.findParentDir is not a function
[quire] ERROR    transforms:format              Eleventy transform error formatting output for _site/bibliography/index.html.
TypeError: import_mockable2.default.findParentDir is not a function
[quire] ERROR    transforms:format              Eleventy transform error formatting output for _site/intro/index.html.
TypeError: import_mockable2.default.findParentDir is not a function
[quire] ERROR    transforms:format              Eleventy transform error formatting output for _site/catalogue/1/index.html.
TypeError: import_mockable2.default.findParentDir is not a function
[quire] ERROR    transforms:format              Eleventy transform error formatting output for _site/catalogue/2/index.html.
TypeError: import_mockable2.default.findParentDir is not a function
→ _site/index.html                     ./content/index.md                   32.1kB --
→ _site/about/index.html               ./content/about.md                   32.6kB --
→ _site/bibliography/index.html        ./content/bibliography.md            32.8kB --
→ _site/catalogue/index.html           ./content/catalogue/index.md         37.9kB --
→ _site/catalogue/1/index.html         ./content/catalogue/1.md             56.3kB --
→ _site/catalogue/2/index.html         ./content/catalogue/2.md             51.1kB --
→ _site/contents/index.html            ./content/contents.md                41.3kB --
→ _site/contributors/index.html        ./content/contributors.md            33.7kB --
→ _site/essay/index.html               ./content/essay.md                   64.6kB --
→ _site/intro/index.html               ./content/intro.md                   42.9kB --
→ _site/pdf-epub-copyr[…]/index.html   ./content/pdf-epub-copyright.md      29.0kB --
→ _site/pdf-epub-half-[…]/index.html   ./content/pdf-epub-half-title.md     27.1kB --
→ _site/pdf-epub-title/index.html      ./content/pdf-epub-title.md          27.3kB --
[11ty] Copied 108 files / Wrote 13 files in 6.62 seconds (509.2ms each, v2.0.1)
[11ty] Watching…
[11ty] Server at http://localhost:8080/
(!) Could not auto-determine entry point from rollupOptions or html files and there are no explicit optimizeDeps.include patterns. Skipping dependency pre-bundling.
```
-------

https://gyazo.com/04bdf7feb947740542423ac304f4bff2

**Reply by geealbers** [reply]
Yes! That's all good. A lot of the messaging there is just informational, the only true errors in all that are known (#728) and as far as we have found, don't have any real consequence.

So, I think you're good to go. I will warn that there is another WIndows error (#626) around `quire build` and `quire pdf` that we're looking at, but I'm glad at least you'll be able to proceed in trying things out with preview working.

**Comment by Erin-Cecele** [comment]
@rachelNelson is also a Windows 11 user. Rachel, I'm tagging you so you can follow along / give input.
